modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue

```diff
@@ -214,10 +195,8 @@ export default {
         if (this.docFields.data.href || e.metaKey) {
```

Comment:
TODO: Does work only for URLs. Should support any type (pages / pieces).
(Might write a ticket for this since this PR is big enough)